circleci.yaml

```diff
@@ -31,6 +31,29 @@ systems:
           This is a generic function to make a circleci API call with the configured token. This
           function is meant to be used for defining other functions.
 
+      add_test_reporter_id:
```

Comment:
Maybe make this a generic function `addEnvVar` that takes the env name as another parameter?